lisa/tools/qemu_img.py

```diff
@@ -1,14 +1,35 @@
 # Copyright (c) Microsoft Corporation.
 # Licensed under the MIT license.
 
+from typing import cast
+
 from lisa.executable import Tool
+from lisa.operating_system import CBLMariner, Linux, Ubuntu
+from lisa.util import LisaException
 
 
 class QemuImg(Tool):
     @property
     def command(self) -> str:
         return "qemu-img"
 
+    @property
+    def can_install(self) -> bool:
+        for _os in [CBLMariner, Ubuntu]:
```

Comment:
`isinstance` is better, because it supports derived system too.

Reply:
doing `isinstance` inside the loop.